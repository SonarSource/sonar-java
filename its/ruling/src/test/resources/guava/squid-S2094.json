{
'com.google.guava:guava:src/com/google/common/reflect/Types.java':[
137,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/GwtFuturesCatchingSpecialization.java':[
27,
],
}
