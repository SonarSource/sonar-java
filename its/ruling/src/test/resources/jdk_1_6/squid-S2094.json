{
'jdk_1_6:java/awt/Component.java':[
452,
],
'jdk_1_6:java/awt/EventQueue.java':[
1041,
],
'jdk_1_6:java/lang/Package.java':[
355,
],
'jdk_1_6:java/lang/Shutdown.java':[
42,
],
'jdk_1_6:java/lang/ref/Reference.java':[
86,
],
'jdk_1_6:java/lang/ref/ReferenceQueue.java':[
35,
],
'jdk_1_6:java/security/ProtectionDomain.java':[
399,
],
}
