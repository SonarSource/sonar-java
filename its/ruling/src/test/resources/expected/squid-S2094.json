{
'project:com/sun/org/apache/xpath/internal/compiler/XPathDumper.java':[
29,
],
'project:java/awt/Component.java':[
473,
],
'project:java/awt/EventQueue.java':[
1226,
],
'project:java/lang/Package.java':[
373,
],
'project:java/lang/Shutdown.java':[
60,
],
'project:java/lang/ref/Reference.java':[
103,
],
'project:java/lang/ref/ReferenceQueue.java':[
52,
],
'project:java/security/ProtectionDomain.java':[
446,
],
'project:javax/xml/bind/JAXBElement.java':[
91,
],
'project:javax/xml/bind/annotation/XmlElement.java':[
209,
],
'project:javax/xml/bind/annotation/XmlElementDecl.java':[
215,
],
'project:javax/xml/bind/annotation/XmlElementRef.java':[
267,
],
'project:javax/xml/bind/annotation/XmlSchemaType.java':[
106,
],
'project:javax/xml/bind/annotation/XmlType.java':[
444,
],
'project:javax/xml/bind/annotation/adapters/XmlJavaTypeAdapter.java':[
114,
],
}
